openshift3/persistent-mysql.json

```diff
@@ -69,19 +69,19 @@
         {
           "name": "CACHE_VOLUME_SIZE",
           "description": "Size of the persistent volume for Redis Cache.",
-          "value": "1Gi",
+          "value": "10Gi",
           "required": true
         },
         {
             "name": "DATABASE_VOLUME_SIZE",
             "description": "Size of the persistent volume for the database.",
-            "value": "1Gi",
+            "value": "40Gi",
```

Comment:
@nijel since we are talking about disk space - what is your experience, what would be average disk usage on database level?

Reply:
The whole https://hosted.weblate.org/ service consumes 8 GB in the database, that's why I think that 40 GB is way too much for default installation. Is this value easy to override when deploying? If yes, I'd stick with original small values, which would allow users to scale up, what is typically way easier than scaling down.

Reply:
OK, lowered it to 10Gi.